src/store/flow/flow.ts

```diff
@@ -201,44 +201,44 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
   async flow_createWith({ dispatch, commit, state }, { props }: {props: {uuid: string} & Partial<IFlow>}): Promise<IFlow> {
     return {
       ...defaults(props, {
-        orgId: '', // awful default value, but we've typed it to string
+        org_id: '', // awful default value, but we've typed it to string
         name: '',
         label: '', // TODO: Remove this optional attribute once the findFlowWith( ) is able to mutate state when setting non existing key.
-        lastModified: moment().format('c'),
-        interactionTimeout: 30,
-        platformMetadata: {},
+        last_modified: moment().format('c'),
+        interaction_timeout: 30,
+        vendor_metadata: {},
 
-        supportedModes: DEFAULT_MODES,
+        supported_modes: DEFAULT_MODES,
         languages: [],
         blocks: [],
 
-        firstBlockId: '',
+        first_block_id: '',
       }),
     }
   },
 
   async flow_duplicateBlock({ dispatch, commit, state }, { flowId, blockId }: {flowId: string; blockId: IBlock['uuid']}): Promise<IBlock> {
-    const flow = findFlowWith(flowId || state.firstFlowId || '', state as unknown as IContext)
+    const flow = findFlowWith(flowId || state.first_flow_id || '', state as unknown as IContext)
     const block: IBlock = findBlockWith(blockId, flow) // @throws ValidationException when block absent
 
     // Deep clone
     const duplicatedBlock = cloneDeep(block)
 
     // Set UUIDs, and remove non relevant props
-    duplicatedBlock.uuid = (new IdGeneratorUuidV4()).generate()
+    duplicatedBlock.uuid = await (new IdGeneratorUuidV4()).generate()
 
-    duplicatedBlock.exits.forEach((item, index, arr) => {
-      item.uuid = (new IdGeneratorUuidV4()).generate()
-      delete item.destinationBlock
+    duplicatedBlock.exits.forEach(async (item, index, arr) => {
+      item.uuid = await (new IdGeneratorUuidV4()).generate()
+      delete item.destination_block
     })
 
     if (has(duplicatedBlock.config, 'prompt')) {
       Vue.set(duplicatedBlock.config, 'prompt', (new IdGeneratorUuidV4()).generate())
     }
 
     // Set UI positions
-    // @ts-ignore TODO: remove this once IBlock has platform_metadata key
-    duplicatedBlock.platform_metadata = {
+    // @ts-ignore TODO: remove this once IBlock has vendor_metadata key
```

Comment:
Still no key even with this update?

Reply:
Unfortunately yes

Reply:
As I know there is no ticket yet for that, I created one https://viamoinc.atlassian.net/browse/VMO-3882